openverse_catalog/dags/common/requester.py

```diff
@@ -38,12 +38,15 @@ class DelayedRequester:
     rate limits of APIs.
 
     Optional Arguments:
-    delay:  an integer giving the minimum number of seconds to wait
-            between consecutive requests via the `get` method.
+    delay:   an integer giving the minimum number of seconds to wait
+             between consecutive requests via the `get` method.
+    headers: a dict that will be passed in all requests, unless overridden
+             by kwargs in specific calls to the get method
     """
 
-    def __init__(self, delay=0):
+    def __init__(self, delay=0, headers={}):
```

Comment:
This looks to be a mutable default argument, which is [definitely something to avoid](https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments) when possible. Can we have this default to `None`, then set `self.headers = headers or {}` below?

Reply:
Ah, right, I forgot about mutations; the void dictionary seemed innocuous. Good catch @AetherUnbound!